src/main.js

```diff
@@ -3,8 +3,10 @@ import App from "./App.vue";
 import router from "./router";
 import store from "./store";
 import { install } from "@/plugins/plugin";
+import ApiService from "./core/api";
```

Comment:
Consider using the consistent path for the entire application.
Webpack configuration in Vue allows the usage of "@" as you've already seen and used and it's consider a best practices.